deploy/tenant-ui/values-int.yaml

```diff
@@ -0,0 +1,29 @@
+fullnameOverride: "int-tenant-ui"
```

Comment:
I assume this file will go once the new deployment structure is implemented, correct?